### DISCUSSION TITLE
How to have the column not visible by default?

### DISCUSSION THREAD

**Discussion by Unisolgr14** [Power-Components/livewire-powergrid#596] [Q&A] [answered]
Firstly, powergrid datatables is a wonderful plugin and I love it.I really appreciate your development team. It reduced my development time. But, I have some issues . How can we make the column invisible not hide and later after the datatable is populated the user will have the option to make it visible through the columntoggle button. And secondly how to give the size of the columns manually(the width).

**Comment by akulmehta** [comment, marked as answer]
Try 

For the hidden:
Add the following hidden method with params as:
```
     ->hidden(true, false),
```

For the styling:
Add the `bodyAttribute` to the Column with parameters

```
   ->bodyAttribute('any-class', 'min-width: 25vw; max-width: 25vw'),
```

**Comment by Unisolgr14** [comment]
Thanks for the quick response. I have applied what you have suggested and its working.

**Reply by akulmehta** [reply]
Great. Happy to help.
